lib/rubocop/ast_node.rb

```diff
@@ -37,8 +37,11 @@ def single_line?
       !multiline?
     end
 
+    COMPARISON_OPERATORS = [:==, :===, :!=, :<=, :>=, :>, :<, :<=>]
```

Comment:
This should be frozen and possibly placed higher up the class def. I don't like mixing constants with method definitions.